### ISSUE TITLE
Button behavior

### ISSUE THREAD

**Issue by ziceptor** [trentech/WirelessRed#2]
Button behavior differs from the default button. Only works if the button is attached directly to the unit (e.g. lamp). If the bulb stands on a stone button on the block of stone will not work. She pressed, but the Redstone signal through the block does not apply. Tool shows that the button is true.

forge-1.10.2-12.18.1.2092-universal.jar
spongeforge-1.10.2-2092-5.0.0-BETA-1756.jar
wirelessred-5.0.0-0.1.0.jar

**Comment by trentech** [comment]
I haven't changed any of the Redstone mechanics for the button. I chose to use button specifically so I wouldn't have to create my own Redstone implementation for receivers.

**Comment by trentech** [comment]
Confirmed. Not sure the cause yet

**Comment by ziceptor** [comment]
[video](https://cloud.mail.ru/public/EvrX/wpgLyb59a)

**Comment by trentech** [comment]
Yeah I've already confirmed the problem. I don't know why it's not working though. Be investigating further

**Comment by ziceptor** [comment]
Any news?

**Comment by trentech** [comment]
Not sure. I think it has something to do with the way Sponge does block updates when changing PoweredData.
